modin/core/storage_formats/pandas/utils.py

```diff
@@ -114,6 +114,61 @@ def split_result_of_axis_func_pandas(
     ]
 
 
+def generate_result_of_axis_func_pandas(
```

Comment:
can we reuse this function in `split_result_of_axis_func_pandas` to avoid code duplication?
```python
def split_result_of_axis_func_pandas(...):
    return list(generate_result_of_axis_func_pandas(...))
```

Reply:
I think this is good, I have added this.